aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Commons.java

```diff
@@ -19,7 +19,8 @@ private Commons() {
     public static final ErrorRuleSet DEFAULT_ERROR_RULE_SET = ErrorRuleSet.builder()
             .withErrorCodes(ErrorStatus.failWith(HandlerErrorCode.ServiceInternalError),
                     ErrorCode.ClientUnavailable,
-                    ErrorCode.InternalFailure)
+                    ErrorCode.InternalFailure,
+                    ErrorCode.InvalidClientTokenId)
```

Comment:
Are we sure that CloudFormation will use new client token in retry?

Reply:
No, but we are silent observers here: we can either retry or bail out immediately. The best we can do is to retry this request and see what happens. Delegating it to the customer with a "invalid token" would certainly cause a "wtf" reaction.